lib/dom.js

```diff
@@ -202,6 +202,12 @@ declare class HTMLCollection {
     [index: number]: Element;
 }
 
+// from http://w3c.github.io/webcomponents/spec/custom/#extensions-to-document-interface-to-register
+declare class ElementRegistrationOptions {
```

Comment:
does this need to be a class, or can it just be an object type, e.g. `type ElementRegistrationOptions = {...`? it doesn't look like you `new` it